lib/tokens.js

```diff
@@ -415,8 +414,16 @@ async function _createToken({
     meta: `z${base58.encode(attributes)}`,
     payload: `z${base58.encode(payload)}`,
   };
-  const token = await cborldEncode(
-    {appContextMap, jsonldDocument, documentLoader});
+  const token = await cborldEncode({
+    appContextMap,
+    jsonldDocument,
+    // FIXME: currently wrapping correct documentLoader function with this,
+    //   just pass in documentLoader when cborld module is fixed
```

Comment:
Can we make this an issue instead? Same for any other FIXMEs that won't be addressed before this PR is merged.

Reply:
Sure thing, opening issue.

Reply:
Thanks. Fixme removed, issue #32 opened.